src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => resolve());
```

Comment:
Consider adding a check to ensure the logo element exists before adding an event listener. This will prevent potential errors if the element is not found in the DOM.